### PR TITLE
Fix/fix create location component/add authorized filters into redux state

### PR DESCRIPTION
**Pull request by mazursasha1990**
Fixed creating location that after adding location automatically changed userData state. Added authorizedFilters into 
redux state

### CODE REVIEW

File: backend/controllers/LocationsController.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ const LocationsController = {
       const searchName = req.query.name as string;
       const filters = JSON.parse(req.query.filters as any);
       const authFilters = JSON.parse(req.query.authFilters as any);
+      console.log('authFilters', authFilters);
+      console.log('filters', filters);
```
Comment (Yojuks) [inline backend/controllers/LocationsController.ts]: why?

File: frontend/src/components/SearchFormContainer/FiltersList/NestedList.tsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default function NestedList() {
   const selectedFilters = useTypedSelector(
     state => state.mapInfo.selectedFilters
   );
+  console.log('selectedFilters', selectedFilters);
```
Comment (Yojuks) [inline frontend/src/components/SearchFormContainer/FiltersList/NestedList.tsx]: why?

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/src/components/SearchFormContainer/FiltersList/NestedList.tsx (hunk 2 above)
Comment (mazursasha1990) [reply frontend/src/components/SearchFormContainer/FiltersList/NestedList.tsx]: Done

File: backend/controllers/LocationsController.ts (hunk 1 above)
Comment (mazursasha1990) [reply backend/controllers/LocationsController.ts]: Done

**Review by Yojuks — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**